migrations/20240606104932_init_metrix.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS public.mtr_metrics (
+	id varchar NOT NULL,
+	mtype varchar NOT NULL,
+	delta bigint NULL,
+	value numeric NULL,
+	CONSTRAINT mtr_metrics_pk PRIMARY KEY (id)
```

Comment:
```suggestion
	CONSTRAINT mtr_metrics_pk PRIMARY KEY (id, mtype)
```
ТЗ не запрещает создавать две записи с одним названием и разными типами.